script_library/clean-airship.sh

```diff
@@ -79,7 +79,7 @@ if [[ ${clean_action} == *"clean_openstack"* ]]; then
     # DO NOT USE clusterrolebinding, else you will delete all rolebindings, even the suse: and system: ones,
     # even when scoped in the namespace.
     kubectl get -n openstack rolebinding.rbac.authorization.k8s.io -o name | xargs kubectl -n openstack delete
-
+    kubectl get jobs -n openstack | awk 'NR >= 1 {print $1 }'| xargs kubectl delete job -n openstack
```

Comment:
why not -o name instead of awk?  Save electrons!

Reply:
haha ..electrons crusader ..liked it.

I guess with -o name, delete command will change as well as I see following output 
``
linux-bybo:/opt # kubectl get jobs -n ucp -o name
job.batch/keystone-bootstrap
job.batch/keystone-credential-setup
``
kubectl get jobs -n openstack -o name | xargs kubectl delete -n openstack